manuscript/manuscript.Rmd

```diff
@@ -545,55 +540,84 @@ In the few replicates that experience a critical transition anyway, the RL agent
+Large, complex problems are still immensely challenging to RL, and existing RL approaches will frequently struggle to solve problems that can already be solved much more efficiently by other means.
+Nevertheless, deep RL approaches using neural network architectures have proven successful on problems that no amount of computational power would put into reach of purely conventional methods.
+Our previous examples have focused on relatively simple examples that can also be addressed at least approximately by conventional methods, and also permit RL-based solutions to be found relatively efficiently on commodity hardware.
+To better illustrate the generality of RL approach and more typical complexity of conservation problems, we now introduce a third example for which we have not yet found an effective policy using RL or classical methods.
 
--->
-```
-<!-- Something about relevance to both the _theory_ and practice of ecological conservation. (e.g. "strategic" 1-D models have probably had a greater influence, (for better or worse!) on real world policies than complex, detail-specific, precise "tactical" models) -->
+<!-- Marcus add ~ paragraph discussion of the fire gym, citing motivating examples, basic overview of the state space, action space and reward and a screenshot of the simulation as a figure? -->
 
-## Open Problems
+# Discussion
 
+<!-- Ecology has complex decisions and we lack necessary tools; RL could be it -->
+Ecological challenges facing the planet today are complex and outcomes are both uncertain and consequential.
+Even our best models and best research will never provide a crystal ball to the future, only better elucidate possible scenarios.
+Consequently, that research must also confront the challenge of making robust, resilient decisions in a changing world.
+The science of ecological management and quantitative decision-making has a long history [e.g. @Schaefer1954; @Walters1974] and remains an active area of research [@Possingham2006; @Fischer2009; @Polasky2011].
+However, the limitations of classical methods such as optimal control frequently constrain applications to relatively simplified models [@Possingham2006], ignoring elements such as spatial or temporal heterogeneity and autocorrelation, stochasticity, imperfect observations, age or state structure or other sources of complexity that are both pervasive and influential on ecological dynamics [@HastingsEncylopedia].
+Complexity comes not only from the ecological processes but also the available actions.
+While conservation objectives are often framed in broad, simple terms, the potential action space is often huge: conservation efforts can be implemented in a wide variety of ways, and the most effective policies may vary in both space and time. <!-- Millie: can we say something more precise about 'simple terms' or add some citation here?-->
```

Comment:
Maybe it's better to just delete the part before the colon, "While conservation objectives are often framed in broad, simple terms, the potential action space is often huge".

I'm tempted to discuss the point that overall objectives aren't particularly complex or dynamic, but I think that's probably a can of worms we don't need to open here.  I'm actually more worried about the reader that doesn't think conservation decisions are as complex as they are currently (i.e. the viewpoint that the science just says 'fish less', 'generate less CO2' -- how to do it isn't up to science'), than the separate issue that current policies operate on artificially simple action spaces (i.e. static in space or time) or oversimplified objective functions (really a different issue from action space entirely).

Reply:
yeah I agree this is an interesting/important point. But also agree that it is maybe an unnecessary can of worms for this piece. If there is space to unpack that simple objectives are a separate issue from overly simple action spaces like you in that comment, it could be valuable? If not, I like your edit of just removing that half of the sentence...